plotly/tools.py

```diff
@@ -2275,6 +2283,62 @@ def create_candlestick(open, high, low, close,
         return dict(data=data, layout=layout)
 
 
+    @staticmethod
+    def create_dendrogram(X, orientation="bottom", labels=None,
+                          colorscale=None):
+        """
+        BETA function that returns a dendrogram Plotly figure object.
+
+        :param (ndarray) X: Matrix of observations as arrray of arrays
+        :param (str) orientation: 'top', 'right', 'bottom', or 'left'
+        :param (list) labels: List of axis category labels(observation labels)
+        :param (list) colorscale: Optional colorscale for dendrogram tree
+        clusters
+
+        X: Heatmap matrix as array of arrays
+        orientation: 'top', 'right', 'bottom', or 'left'
+        labels: List of axis category labels
+        colorscale: Optional colorscale for dendrogram tree clusters
+
+
+        Example 1: Simple bottom oriented dendrogram
+        ```
+        import plotly.plotly as py
+        from plotly.tools import FigureFactory as FF
+
+        import numpy as np
+
+        X = np.random.rand(5,5)
+        dendro_X = FF.create_dendrogram(X)
```

Comment:
IMO, the `dendro_X` variable seems like it would be clearer to read as a user if just `dendro`? Super minor, and maybe it's my misunderstanding.